src/components/Match/MatchLog.jsx

```diff
@@ -96,7 +96,11 @@ const logColumns = [heroTdColumn, {
     switch (row.type) {
       case 'kills': {
         const hero = heroNames[row.detail] || {};
-        return <img src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />;
+        const comp = [<img key="hero" src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />];
```

Comment:
Actually, I'mt not sure what you're trying to do here.  If you're trying to show some string if row.tracked_death is true, you should be able to draw another element in the div.  Having an array of an img element and a string seems kind of inconsistent